languages/csharp/reference/exercise-concepts/bowling.md

```diff
@@ -0,0 +1,35 @@
+# Concepts of bowling
+
```

Comment:
Could you add the link to the example implementation in v2 please?

Reply:
Done!